docs/refs/api/catalogd-webserver.md

```diff
@@ -0,0 +1,88 @@
+# catalogd webserver
```

Comment:
```suggestion
# Catalogd web server
```

Reply:
I notice that both _webserver_ and _web server_ are used. Downstream, we use [_web server_](https://redhat-documentation.github.io/supplementary-style-guide/#web-server), so that is what I suggested here. 

If we consistently use _webserver_ right now, I suggest with stick with that for now.

Reply:
Ah I tried to update all instances to "web server" but looks like I missed this one

Reply:
Should be all "web server" as of https://github.com/operator-framework/operator-controller/pull/1331/commits/de1767ea0edfa05872092fc0aee3dda0616bd305